File: C/Array/Basic/misc.java
class Demo {

	public static void main(String[] args) {

		//char c = 14.1;	//Error
		//int a4 = 41l;		//Error
		//int a2 = 75.4;	//Error


		double d = 41.2f;
		float g = 10;

		int a = 'A';
		char c1 = 65;
		System.out.println(c1 + " " + a);

		char[] array1 = new char[] {97, 98, 99};

		for(char c2 : array1)
			System.out.print(c2 + " ");
		System.out.println();

		//int[] array2 = new int[] {14, 89.1};

		//fun(70);	//Error
		//gun(100.5);	//Error
	}

	static void fun(char a) {

	}

	static void gun(int b) {

	}
}

File: C/Array/Basic/misc.c
#include<stdio.h>

void gun(int a) {

	printf("%d in gun\n", a);
}

void fun(char c) {
	
	printf("%c in fun\n", c);
}

void main() {
	
	int a1 = 75.4;
	printf("%d\n", a1);
	
	char c = 65.11;
	printf("%c\n", c);

	int a = 41l;
	printf("%d\n", a);

	double d = 41.2f;
	printf("%lf\n", d);

	float g = 10;
	printf("%f\n", g);

	int b = 'Z';
	printf("%d\n", b);

	char ch = 65;
	printf("%c\n", ch);

	fun(70);
	gun(70.2);
}
